src/current/releases/molt.md

```diff
@@ -18,11 +18,20 @@ To download the latest MOLT Fetch/Verify binary:
 
 {% include molt/molt-install.md %}
 
+## August 15, 2024
+
+MOLT Fetch/Verify 1.1.5 is [available](#installation).
+
+- Deprecated the `--ongoing-replication` flag in favor of [`--mode data-load-and-replication`]({% link molt/molt-fetch.md %}#load-data-and-replicate-changes), using the new `--mode` flag. Users should replace all instances of `--ongoing-replication` with `--mode data-load-and-replication`.
```

Comment:
Is there a way to maybe bold this point around deprecation so it stand out more?

Reply:
Bolded!